config.go

```diff
@@ -385,15 +422,15 @@ func (config *TGFConfig) validate() (errors []error) {
 
 	if config.RecommendedTGFVersion != "" && version != locallyBuilt {
 		if valid, err := CheckVersionRange(version, config.RecommendedTGFVersion); err != nil {
-			errors = append(errors, fmt.Errorf("Unable to check recommended tgf version %s vs %s: %v", version, config.RecommendedTGFVersion, err))
+			errors = append(errors, fmt.Errorf("unable to check recommended tgf version %s vs %s: %v", version, config.RecommendedTGFVersion, err))
```

Comment:
There was a warnings coming from `go`'s compiler saying that errors should not start with capitals. I just made the warning go away. There's 3 more instances later of this. It's all for the same reason